docs/api.md

```diff
@@ -173,6 +173,16 @@ Keep the subprocess alive while `getEachMessage()` is waiting.
 
 [More info.](ipc.md#keeping-the-subprocess-alive)
 
+### getCancelSignal()
+
+_Returns_: [`Promise<AbortSignal>`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal)
+
+Retrieves the [`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal) shared by the [`cancelSignal`](#optionscancelsignal) option.
+
+This requires the [`gracefulCancel`](#optionsgracefulcancel) option to be `true`.
+
```

Comment:
Added the [following line](https://github.com/sindresorhus/execa/blob/graceful-cancel/docs/api.md#getcancelsignal):

> This can only be called inside a subprocess.